modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/product/navigation/control/menu/AnalyticsReportsProductNavigationControlMenuEntry.java

```diff
@@ -182,6 +183,16 @@ public boolean includeIcon(
 	public boolean isShow(HttpServletRequest httpServletRequest)
 		throws PortalException {
 
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		if (FeatureFlagManagerUtil.isEnabled(
```

Comment:
A minor question.
This FF here should be enabled or disabled. I'm confused

Reply:
This is really for checking if feature flag is enabled. Because we are moving the sidebase this button shows to the new tab created in content dashboard sidebar

Reply:
So when the FF is enable this  button should not appear and the new tab in content dashboard sidebar should.